src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Windows.cs

```diff
@@ -279,18 +283,40 @@ protected virtual void UpdateEmptyView()
 
 		protected virtual void UpdateItemsLayout()
 		{
-			ListViewBase.IsSynchronizedWithCurrentItem = false;
+			if (ListViewBase is FormsGridView gridView)
+			{
+				if (Layout is LinearItemsLayout linearItemsLayout)
+				{
+					gridView.Orientation = linearItemsLayout.Orientation == ItemsLayoutOrientation.Horizontal
+						? Orientation.Horizontal
+						: Orientation.Vertical;
 
-			FindScrollViewer(ListViewBase);
+					gridView.Span = 1;
 
-			_defaultHorizontalScrollVisibility = null;
-			_defaultVerticalScrollVisibility = null;
+					if (linearItemsLayout.ItemSpacing != _previousItemSpacing)
+					{
+						_previousItemSpacing = linearItemsLayout.ItemSpacing;
+						gridView.ItemContainerStyle = linearItemsLayout.GetItemContainerStyle();
+					}
+				}
 
-			UpdateItemTemplate();
-			UpdateItemsSource();
-			UpdateVerticalScrollBarVisibility();
-			UpdateHorizontalScrollBarVisibility();
-			UpdateEmptyView();
+				if (Layout is GridItemsLayout gridItemsLayout)
+				{
+					gridView.Orientation = gridItemsLayout.Orientation == ItemsLayoutOrientation.Horizontal
+						? Orientation.Horizontal
+						: Orientation.Vertical;
```

Comment:
WE have code duplication here, add an extension method that does the ItemsLayoutOrientation to Orientation conversion, probably a good API to have to be used internally.